### ISSUE TITLE
Support NEO-GEO SNK games

### ISSUE THREAD

**Issue by sigboe** [sigboe/pie-galaxy#4]
Support neogeo games. slated for 0.3 release.

**Comment by sigboe** [comment]
Unfinished implementation in the next branch from commit 4e498e40b1e8b30838782ef0a4309c139f320f6b

**Comment by sigboe** [comment]
@MajorDangerNine
0636b5955d82f02b19a8aa3da44f17a57d5fca38
This commit should allow you to download the Linux installers for the games that only have them in the extras. Can you test

**Comment by MajorDangerNine** [comment]
**Extras Download Works, Installs, and Works**

- Garou: Mark of the Wolves
- Last Blade, The
- Metal Slug
- Shock Troopers
- Shock Troopers: 2nd Squad
- Twinkle Star Sprites

**Extras Download Doesn't Work**

- Baseball Stars 2
- Metal Slug 2
- Metal Slug 3
- Metal Slug X

This is an example of the error for each game.
```
wyvern: Found no games named METAL SLUG 2 in your library.
```
Logs: [Neo Geo - Failed Extras Download Logs.zip](https://github.com/sigboe/pie-galaxy/files/3035874/Neo.Geo.-.Failed.Extras.Download.Logs.zip)

**Notes**
This fix by downloading all Extras will be wasteful of space for the end-user since they cannot view the other files Pie Galaxy downloads to delete them. In the case of these games, the files downloaded are the OSX installations.

I advise a Clear option be added to the main menu that will allow the end-user to quickly clear out the Downloads folder (with confirmation of course).

**Comment by sigboe** [comment]
@MajorDangerNine an update on this, since I have been travelling for a while and I thought I wrote here before I did.

I have a private branch that depends on the newest version of wyvern that will let you pick between the extras to download. 

![image](https://user-images.githubusercontent.com/2087930/56564365-24586b00-65ae-11e9-83f6-4ccf776785cf.png)

@nicohman has told me that he intend to do this sometime soon so we will wait patiently, and I will close this issue when I push the change that depends on his :)

This should fix both the extras that does not download, and allow you to download  only that which you want. 

@MajorDangerNine as for deleting files, do the extras that are game installers just download as raw/direct installer files, or do they come zipped and require you to unzip before install? The filebrowser in the installer of Pie Galaxy supports deleting files, but will only show `.exe` and `.sh` files. I have been thinking about adding a pdf manual viewer, but that is just in the thinking stage yet.
